src/module/Task.js

```diff
@@ -0,0 +1,28 @@
+class Task {
+  constructor() {
+    if (JSON.parse(localStorage.getItem('TASKS_LIST')) != null) {
+      this.TaskObject = JSON.parse(localStorage.getItem('TASKS_LIST'));
+    } else {
+      this.TaskObject = [];
+    }
+  }
+
+  add(task) {
+    this.TaskObject.push({
+      id: this.TaskObject.length + 1,
+      description: task.value,
+      completed: false,
+    });
+    localStorage.setItem('TASKS_LIST', JSON.stringify(this.TaskObject));
+  }
+
+  remove(element) {
+    if (element.classList.contains('taskDynamic')) {
+      element.remove();
+      this.TaskObject.splice(element.getAttribute('data-id'), 1);
+    }
+    localStorage.setItem('TASKS_LIST', JSON.stringify(this.TaskObject));
+  }
```

Comment:
- [ ] I have done several attempts to delete a task but to no avail. Please make sure to fix this.

Reply:
@FrederickMih 
This project is complete!
Tasks are deleting effectively when I reviewed them.

Reply:
@rloterh, I have tested your project over and over

Reply:
Thank you @FrederickMih for the review.
I corrected the error, kindly review to enable me proceed